src/main/scala/com/JonathanMa/main.scala

```diff
@@ -24,9 +26,37 @@ object main {
     println("The factored version of 64/128 is: " + rational4)// result of using a private val and method
     println("Rational " + rational4 + " + " + integer + " = " + (rational4+integer))
      */
-//    grep("random")
-//    tryCatching()
-//    println(multiTable())
+    //    grep("random")
+    //    tryCatching()
+    //    println(multiTable())
 
+    object Spiral {
+      val space = elem(" ")
+      val corner = elem("+")
+
```

Comment:
it'll be helpful to add comments to demo the print out of this code, it can help others who read it